src/components/pages/dashboard/dashboard.tsx

```diff
@@ -51,9 +51,14 @@ const Dashboard: FC = () => {
     }
   };
 
-  const getToUrlForArtifact = (_: EntityType, id: string): string => {
+  const getToUrlForArtifact = (entityType: EntityType, id: string): string => {
     const cdaArtefactUrlPattern = import.meta.env.VITE_CDA_ARTEFACT_URL as string;
-    return cdaArtefactUrlPattern.replace('{{lang}}', ui.lang).replace('{{id}}', id);
+
+    const artefactId = entityType === EntityType.LITERATURE_REFERENCE
+      ? `literature-${id}`
```

Comment:
Wäre es nicht besser, wenn das Präfix `literature-` mit in den Daten steht? Dann bräuchte man hier keine Fallunterscheidung.

Reply:
Da können wir hier leider nicht viel machen. Die Seiten liegen auf dem Server unter diesem Namen.